tests/contrib/pylibmc/test.py

```diff
@@ -77,6 +80,32 @@ def test_incr_decr(self):
         resources = sorted(s.resource for s in spans)
         eq_(expected_resources, resources)
 
+    def test_incr_decr_ot(self):
+        """OpenTracing version of test_incr_decr."""
+        client, tracer = self.get_client()
+        ot_tracer = init_tracer('memcached', tracer)
+
+        start = time.time()
+        with ot_tracer.start_active_span('mc_ops'):
+            client.set("a", 1)
+            client.incr("a", 2)
+            client.decr("a", 1)
+            v = client.get("a")
+            assert v == 2
```

Comment:
[nit-picking]: it **does not** prevent merge.
For consistency with the majority of our tests, shouldn't we use `eq_(2, v)` in place of `assert v == 2`.